packages/ts/generator-plugin-signals/test/fixtures/ValueSignalServiceMix.snap.ts

```diff
@@ -6,7 +6,9 @@ async function getPerson_1(init?: EndpointRequestInit_1): Promise<Person_1 | und
 function personSignal_1(): ValueSignal_1<Person_1 | undefined> {
     return new ValueSignal_1(undefined, { client: client_1, endpoint: "PersonService", method: "personSignal" });
 }
-function personSignalNonNull_1(): ValueSignal_1<Person_1> {
-    return new ValueSignal_1(undefined, { client: client_1, endpoint: "PersonService", method: "personSignalNonNull" });
+function personSignalNonNull_1({ defaultValue: defaultValue_1 }: {
```

Comment:
Could we also have a test function that validates the combination of a default client-side value and the service params work together?

Reply:
Good catch! It needs a fix indeed.